tap_intercom/stream.py

```diff
@@ -54,6 +59,14 @@ def do_sync(self, tap_stream_id: str, state: Optional[datetime] = None):
                         tap_stream_id=tap_stream_id,
                     )
                     latest_bookmark = replication_value
+            except requests.exceptions.HTTPError as e:
+                data = e.response.json()
+                errors = data.get("errors", [])
+                code = errors[0].get("code")
+                message = errors[0].get("message")
```

Comment:
Note that these will go out of bounds if `errors` ends up being `[]`.
If we're okay with that, then that's cool.

Reply:
Yup I actually want to fix that